src/settingsholder.cpp

```diff
@@ -14,7 +14,8 @@ constexpr bool SETTINGS_LOCALNETWORKACCESS_DEFAULT = false;
 constexpr bool SETTINGS_UNSECUREDNETWORKALERT_DEFAULT = false;
 constexpr bool SETTINGS_CAPTIVEPORTALALERT_DEFAULT = false;
 constexpr bool SETTINGS_STARTATBOOT_DEFAULT = false;
-constexpr bool SETTINGS_PROTECTSELECTEDAPPS_DEFAULT = true;
+constexpr bool SETTINGS_PROTECTSELECTEDAPPS_DEFAULT = false;
```

Comment:
Also i forgot that SETTINGS_PROTECTSELECTEDAPPS_DEFAULT should have not been active by default.